walkbox.go

```diff
@@ -173,3 +173,20 @@ func (wm *WalkBoxMatrix) EnableWalkBox(id int, enabled bool) {
 		wm.itineraryMatrix = calculateItineraryMatrix(wm.walkBoxes)
 	}
 }
+
+// WalkBoxAt returns the walk box identifier at the given position or the closest one,
+// along with a boolean indicating inclusion.
+func (wm *WalkBoxMatrix) WalkBoxAt(p *Positionf) (id int, included bool) {
```

Comment:
Considering this is an operation of the walkbox matrix, and the matrix stores indices, the returned integer is not an ID but the index of the box. Considering this index corresponds to the i-th position of the walkboxes slice where the box can be found. 

However, I don't get why this operation and the rest are done in the matrix itself. In my mind, the matrix is just a bi-dimensional table of box indexes. I would expect it lacks the information to know what is the box closest to p.

Reply:
Ok, forget above comment. After a deep review of #21, I see the intention is to store the boxes into the matrix and add it the responsibility of knowing about the boxes and their relations to it. I don't fully like the idea, but let's go with it for the moment.  

However, I don't see how this function would be used out from `WalkBoxMatrix`. It returns a box index, that is somehow useless out from the matrix and the slice of boxes it contains privately. If this is just an auxiliary operation to have a public `FindPath(a, b Position) []Position`, then it should be non-exported (private). 

Same comment applies to the rest of operations defined here.

Reply:
Make a lot of sense, lets do it!